src/meta/app/src/schema/table.rs

```diff
@@ -643,7 +643,7 @@ pub struct GetTableCopiedFileReply {
 
 #[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, Eq)]
 pub struct UpsertTableCopiedFileReq {
-    pub table: TableNameIdent,
+    pub table_id: u64,
```

Comment:
For GetTableCopiedFileReq, UpsertTableCopiedFileReq and TruncateTableReq, to use table id, it still needs a `seq` to identify a specific version of table, i.e., use `TableIdent`.

Reply:
@BohuTANG does the table copied files meta need to precise to table version? Or just to table id is enough?

If it needs to precise to table version, it means that query layer need some transaction too:

1. get the table id and seq.
2. operate with table id and seq.
3. if get table seq in meta is not match to the seq passed, goto step 1 and try again...

Reply:
add `TableCopiedFileLock` to make sure consistent.